rules/build_test.bzl

```diff
@@ -16,16 +16,40 @@
 
 load("//lib:new_sets.bzl", "sets")
 
+def _empty_test_impl(ctx):
+    is_windows = select({
+        "@bazel_tools//src/conditions:host_windows": True,
+        "//conditions:default": False,
+    })
+    extension = ".bat" if is_windows else ".sh"
```

Comment:
```suggestion
    extension = ".bat" if ctx.attr.is_windows else ".sh"
```

Reply:
This is a bug, the select is not evaluated as desired in this context.
However it is truthy, therfore the if does parse. Therefore the condition always evaluates to `True` and extension .bat is always used.